cordova-common/src/ConfigChanges/ConfigFile.js

```diff
@@ -17,13 +17,30 @@
 var fs = require('fs');
 var path = require('path');
 
-var bplist = require('bplist-parser');
-var et   = require('elementtree');
-var glob = require('glob');
-var plist = require('plist');
+var modules = {};
+
+function addProperty(obj, property, modulePath) {
```

Comment:
Perhaps add an optional 4th parameter which is the name of a property on the required module.